src/perl/Permabit/BlockDevice/VDO.pm

```diff
@@ -600,20 +609,26 @@ sub installModule {
                                   modName         => $moduleName,
                                   modVersion      => $version,
                                   useDistribution => $self->{useDistribution},
+                                  useDnf          => $self->{useLinuxNextModule},
                                  );
   $module->load();
   $self->{_modules}{$machineName}{$moduleName} = $module;
   $self->_setModuleParameters();
 
-  $log->info("Installing module $VDO_USER_MODNAME $version on $machineName");
+  my $modVersion = $self->{useLinuxNextModule} ? $VDO_LINUXNEXT_MODVER
+                                               : $self->getModuleVersion();
+
   my $userModule
     = Permabit::UserModule->new(
                                 machine         => $self->getMachine(),
                                 modDir          => $self->getModuleSourceDir(),
                                 modName         => $VDO_USER_MODNAME,
-                                modVersion      => $version,
+                                modVersion      => $self->{useLinuxNextModule}
+                                                    ? $VDO_LINUXNEXT_MODVER
+                                                    : $self->getModuleVersion(),
                                 useDistribution => $self->{useDistribution},
-                               );
+                                useDnf          => $self->{useLinuxNextModule},
+                                );
   $userModule->load();
```

Comment:
Same comments as above, just pointing you'll need to adjust these arguments also.